src/main/resources/assets/thebetweenlands/lang/zh_CN.lang

```diff
@@ -1070,19 +1070,26 @@ command.blevent.failure.alreadydisabled=天气事件已经被禁用了。
 command.blevent.failure.on=无法使天气事件 '%s' 开始。
 command.blevent.failure.off=无法使天气事件 '%s' 结束。
 
+command.aspectdiscovery.usage=指令格式: /aspectDiscovery <玩家名> <reset|discover> <held|all>
 command.aspectdiscovery.book.none=没有药学手册可供书写。
 command.aspectdiscovery.held.null=你必须拿着一个物品。
 command.aspectdiscovery.discover.held=结果: %s 要素: %d
 command.aspectdiscovery.discover.all=发现所有物品的所有要素。
 command.aspectdiscovery.reset.held=清除所有已经发现的要素。
 command.aspectdiscovery.reset.all=清除所有物品的已发现要素。
 
+command.aspect.reset.usage=指令格式: /resetAspects
 command.aspect.reset.confirm=输入 '/resetAspects confirm'以确认。
 command.aspect.reset.success=重设要素完成。
 
+command.blreloadrecipes.usage=指令格式: /blReloadRecipes
 command.blreloadrecipes.success=重新载入自定义配方。
 command.blreloadrecipes.failed=载入自定义配方失败。
 
+command.set_decay.usage=指令格式: /setDecay <player> <decay> [saturation]
```

Comment:
You may want to replace <player> here with the same word you used for /aspectDiscovery. <decay> and [saturation] are also variables so feel free to translate those too.